src/components/AnonymousChat/child/ChannelPostNotificationMessage.tsx

```diff
@@ -1,47 +1,47 @@
 import * as React from 'react';
 import {StyleSheet, Text} from 'react-native';
 
-import useProfileHook from '../../../hooks/core/profile/useProfileHook';
 import {BaseChannelItemTypeProps} from '../../../../types/component/AnonymousChat/BaseChannelItem.types';
 import {colors} from '../../../utils/colors';
-import {fonts} from '../../../utils/fonts';
+import {fonts, normalize} from '../../../utils/fonts';
 
 const ChannelPostNotificationMessage = ({
   type = BaseChannelItemTypeProps.ANON_PM,
-  commenterId,
   commenterName,
   message
 }) => {
-  const {profile} = useProfileHook();
-  if (type === BaseChannelItemTypeProps.ANON_PM) return <></>;
-
   const styles = StyleSheet.create({
     chatContentPostNotificationMessage: {
       fontFamily: fonts.inter[400],
-      fontSize: 12,
+      fontSize: normalize(12),
       lineHeight: 20,
       color: colors.gray,
       marginRight: 29
     },
     chatContentPostNotificationMessageBold: {
       fontFamily: fonts.inter[600],
-      fontSize: 12,
+      fontSize: normalize(12),
       lineHeight: 20,
       color: colors.gray
     }
   });
+
+  const isPM = type?.includes('PM');
+  const amICommenter = type?.includes('I_COMMENTED');
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `includes` method is used here which might lead to incorrect results if the `type` string contains 'PM' or 'I_COMMENTED' in any part of it, not just as separate words. Consider using a more precise comparison.



<!-- This is an auto-generated comment by OSS CodeRabbit -->